compiler/luci/pass/include/luci/Pass/ResolveCustomOpPass.h

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2023 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __LUCI_RESOLVE_CUSTOM_OP_PASS_H__
+#define __LUCI_RESOLVE_CUSTOM_OP_PASS_H__
+
+#include <logo/Pass.h>
+
+namespace luci
+{
+
+/**
+ * @brief Class to convert a custom operator to a built-in operator.
+ * pass to which ops that used to be custom op but changed to builtin ops will be applied.
+ */
+struct ResolveCustomOpPass final : public logo::Pass
+{
+  ResolveCustomOpPass() {}
+
+  const char *name(void) const final { return "luci::ResolveCustomOpPass"; }
+
+  bool run(loco::Graph *g) final;
+};
+
+} // namespace luci
+
+#endif // __LUCI_RESOLVE_CUSTOM_OP_PASS_H__
```

Comment:
The overall description of the new pass is as follows.

- The discussion of the New pass, `ResovleCustomOpPass`, came from here.
-> https://github.com/Samsung/ONE/issues/10784#issuecomment-1797862821
- This new pass is a pass that changes op, which was previously supported by `custom` op, 
when it can now be supported by `built-in` op.